src/components/search-box.vue

```diff
@@ -1,26 +1,27 @@
 <template>
-  <div class="x-input-group x-input-group--card x-border-color--lead">
+  <div class="x-input-group x-input-group-lead x-rounded-sm">
```

Comment:
I don't know if the designs are updated, but there the input group has the neutral 90 color.

```suggestion
  <div class="x-input-group x-rounded-sm">
```

Reply:
the designs we have to match are these of the XDS: https://www.figma.com/file/9a1DMgqIHt5ZZBvS2waaOs/X-DS?node-id=4126%3A81277&t=mcdh9xFjIwKM0MlH-0